generic/openblas.py

```diff
@@ -49,14 +50,24 @@ def setUp(self):
         openblas_dir = os.path.join(self.workdir, "OpenBLAS-develop")
         openblas_bin_dir = os.path.join(openblas_dir, 'bin')
         os.mkdir(openblas_bin_dir)
-        build.make(openblas_dir, extra_args='FC=gfortran '
-                   'ISMAXKERNEL=imax.S ISMINKERNEL=imin.S')
+        cmd = "gcc --version"
+        output = process.system_output(cmd, shell=True).decode("utf-8")
+        gcc_ver = re.search(r'\d+', output)
+        proc_type = cpu.get_family()
+        self.target = ""
+        if int(gcc_ver.group()) < 10 and 'power10' in proc_type:
+            self.target = "TARGET=POWER9"
+            build.make(openblas_dir, extra_args='FC=gfortran '
+                       'ISMAXKERNEL=imax.S ISMINKERNEL=imin.S TARGET=POWER9')
+        else:
+            build.make(openblas_dir, extra_args='FC=gfortran '
+                       'ISMAXKERNEL=imax.S ISMINKERNEL=imin.S')
         build.make(openblas_dir, extra_args='PREFIX=%s install' %
                    openblas_bin_dir)
         self.test_dir = os.path.join(openblas_dir, "test")
 
     def test(self):
-        result = build. run_make(self.test_dir)
+        result = build.make(self.test_dir, extra_args=self.target)
```

Comment:
`build.make` returns only exit status, hence the following lines after this wont make sense. So please change it accordingly to `run_make`.